### DISCUSSION TITLE
Assessment service is not running

### DISCUSSION THREAD

**Discussion by shubham54-sca** [Sunbird-Knowlg/Community#136] [Bugs]
I have execute "mvn play2: run" command to run the service in Directory = "/knowledge-platform/assessment-api/assessment-service/" but getting below error.

[ERROR] 1 required artifact is missing.
[ERROR]
[ERROR] for artifact:
[ERROR] org.sunbird:assessment-actors:jar:1.0-SNAPSHOT
could you guys pls help me to solve this issue? 
other services in the knowledge building block are working fine.

@AmiableAnil @vinukumar-vs

**Comment by pallakartheekreddy** [comment]
@shubham54-sca assessment-service now part of Inquiry. Please raise this issue in the below discussion form.
https://github.com/orgs/Sunbird-inQuiry/discussions

@sajeshkayyath FYI

**Reply by sajeshkayyath** [reply]
cc @PoojaJManjooran / @krgauraw
